### DISCUSSION TITLE
problem loading quick start templates

### DISCUSSION THREAD

**Discussion by abhibollam** [Budibase/budibase#9781] [Help]
Hi Team,

is there any way to turn off tis error when you are behind a company proxy?

it keeps on popping up for every click

![image](https://user-images.githubusercontent.com/126145059/220842030-e9dd55d2-9a2d-44ea-b6b1-bb40afb85dea.png)

**Comment by andz-bb** [comment]
hey @abhibollam, unfortunately we don't have a way to disable these notifications in the builder. If you have a way to modify the Budibase code for your instance there's a solution in this discussion: #5327

**Reply by abhibollam** [reply]
can you please help me with this as well https://github.com/Budibase/budibase/discussions/9780

i'm trying to find out the issue from 3 days going no where

**Comment by ksorgIN** [comment]
I just installed for a local install and am receiving the same error.  Why can we not get the templates on a on prem install? 
![image](https://user-images.githubusercontent.com/127224118/223463470-c813288c-8692-466b-ad52-5eadb8a9b87d.png)
